tests/staticfiles_tests/test_storage.py

```diff
@@ -445,6 +445,13 @@ def test_missing_entry(self):
         # File exists on disk
         self.hashed_file_path(missing_file_name)
 
+    def test_doesnt_keep_intermediate_files(self):
+        intermediate_file = 'cached/relative.ccc37118948a.css'
+        collected_path = os.path.join(settings.STATIC_ROOT, intermediate_file)
+
+        # Intermediate file wasn't written to disk
+        self.assertFalse(os.path.exists(collected_path))
```

Comment:
This test is passing even without a fix.

Reply:
I temporarily split this PR into two commits to run this test independently on Jenkins and it looks like you're right. However when I checkout d07361856e locally and run the tests it does fail.
I'm not sure what it is happening here.

I have two questions to help me look into this:
- Where can I find the Jenkins config that runs on djangoci.com? So I can compare it to my local `./runtests.py`
- How did you detect that this test was passing without a fix? Does it succeed on your computer?

Reply:
Jenkins doesn't do anything specific, and yes this test passes locally for me without a fix.